.travis-build-without-test.sh

```diff
@@ -6,11 +6,21 @@ set -e
 
 export SHELLOPTS
 
+
```

Comment:
The following is shorter and more direct, and could be used elsewhere:

```
SLUGOWNER=${TRAVIS_REPO_SLUG%/*}
SLUGREPO=${TRAVIS_REPO_SLUG##*/}
```

Duplicating those 2 lines (or just the one that is necessary) seems better than duplicating 7 lines.